plugin/storage/es/esRollover.py

```diff
@@ -132,6 +134,18 @@ def create_aliases(client, alias_name, archive_index_name):
     alias.do_action()
 
 
+def alias_is_not_empty(client, alias_name):
```

Comment:
Shouldn't this be called `alias_is_empty`?

Reply:
yes, apart from that is it OK?

Reply:
Yes LGTM.